src/providers/noop.rs

```diff
@@ -36,9 +36,9 @@ impl PushProvider for NoopProvider {
 impl NoopProvider {
     /// Insert empty notifications for a new token
     fn bootstrap(&mut self, token: String) {
-        if !self.notifications.contains_key(&token) {
-            self.notifications.insert(token, vec![]);
-        }
+        self.notifications
+            .try_insert(token, vec![])
```

Comment:
Is this what the nightly for? And why change this code anyway?

Reply:
Clippy dosen't like the if not exists, then insert

Reply:
Fixed